x-pack/plugins/cases/server/routes/api/utils.test.ts

```diff
@@ -5,317 +5,10 @@
  * 2.0.
  */
 
-import {
-  transformNewCase,
-  transformNewComment,
-  wrapError,
-  transformCases,
-  flattenCaseSavedObject,
-  flattenCommentSavedObjects,
-  transformComments,
-  flattenCommentSavedObject,
-  sortToSnake,
-} from './utils';
-import { newCase } from './__mocks__/request_responses';
+import { wrapError } from './utils';
```

Comment:
Most of the functionality was moved to `common/utils` or `client/utils` because the files under `routes/api` no longer reference these functions.